### PR TITLE
[graphics] Write documentation for graphics

### PR DESCRIPTION
**Pull request by GauchKirill**
- Also corrected spelling mistakes

### CODE REVIEW

File: include/graphics/button.hpp [hunk 1]
```diff
@@ -23,19 +25,36 @@ class Button {
...
+    // Sets the position of the text
+    void set_text_position();
 
   public:
     Button();
     Button(const std::string &text, const sf::IntRect &rect = sf::IntRect());
     ~Button();
-    void set_possition(const sf::IntRect &rect);
+
+    // Set buttun position
```
Comment (aleksplast) [inline include/graphics/button.hpp]: button

File: include/graphics/button.hpp [hunk 2]
```diff
@@ -23,19 +25,36 @@ class Button {
...
+    // Set text color
     void set_text_color(sf::Uint8 red, sf::Uint8 green, sf::Uint8 blue, sf::Uint8 alpha);
+
+    // Set text color
     void set_text_color(const sf::Color &color);
+
+    // Draw button on the window
     void draw(sf::RenderWindow &window);
+
+    // Return buttun global bounds
```
Comment (aleksplast) [inline include/graphics/button.hpp]: button

File: include/graphics/command.hpp [hunk 3]
```diff
@@ -13,8 +13,10 @@ class PlayerCard;
 class Base;
 template <typename T> class Deck;
 
+// A class that handles game events
 class Command {
   private:
+    // enum class Type
```
Comment (aleksplast) [inline include/graphics/command.hpp]: Types of what?

File: include/graphics/button.hpp [hunk 4]
```diff
@@ -9,8 +9,10 @@
 
 namespace Mayhem {
 
+// A class describing the functions of the button
 class Button {
   public:
+    // enum class Type
```
Comment (aleksplast) [inline include/graphics/button.hpp:15]: Types of what?

File: include/graphics/command.hpp [hunk 5]
```diff
@@ -45,19 +48,37 @@ class Command {
     GraphicsModel &model;
 
   private:
+    // determines the status of the team
```
Comment (aleksplast) [inline include/graphics/command.hpp]: What is the `team`

File: include/graphics/command.hpp [hunk 6]
```diff
@@ -45,19 +48,37 @@ class Command {
...
     template <int N> Status is_this_command(const std::array<Type, N> &data) const;
+
+    // checks whether the command can be executed
     void check_commands();
+
+    // Move action is playing
```
Comment (aleksplast) [inline include/graphics/command.hpp]: better to write `Play move action`

File: include/graphics/command.hpp [hunk 7]
```diff
@@ -45,19 +48,37 @@ class Command {
...
     void activate_move_action();
+
+    // Typical action is playing
```
Comment (aleksplast) [inline include/graphics/command.hpp]: Same as above

File: include/graphics/drawable.hpp [hunk 8]
```diff
@@ -16,9 +17,16 @@ class Drawable {
...
     Drawable() = default;
     Drawable(const std::string &texture_file);
+
+    // Draw entity on the window
     virtual void draw(sf::RenderWindow &window, const sf::FloatRect &rect, const float angle);
+
+    // clicks on the entity
     void press();
+
+    // release the entity
```
Comment (aleksplast) [inline include/graphics/drawable.hpp]: release from what

File: include/graphics/graphics_model.hpp [hunk 9]
```diff
@@ -10,35 +10,52 @@ class PlayerCard;
...
+// The graphics class responsible for data storage
 class GraphicsModel {
   public:
+    // Graphics settings
     struct Settings {
+
+        // Game types
         enum class GameType {
             online,
             ofline,
```
Comment (aleksplast) [inline include/graphics/graphics_model.hpp]: offline

File: include/graphics/graphics_model.hpp [hunk 10]
```diff
@@ -10,35 +10,52 @@ class PlayerCard;
...
+            // Drawing Player
             struct Player {
+
+                // Scale
                 struct Scale {
                     static constexpr float x = 0.38;
                     static constexpr float y = 0.25;
                 }; // struct Slace
 
+                // increase of the main player
```
Comment (aleksplast) [inline include/graphics/graphics_model.hpp]: increase of what

File: include/graphics/graphics_model.hpp [hunk 11]
```diff
@@ -88,13 +119,19 @@ class GraphicsModel {
 
             }; // strust BasesPlace
 
+            // Drawing Buttuns
```
Comment (aleksplast) [inline include/graphics/graphics_model.hpp]: buttons

**Review by aleksplast — COMMENTED** [review]
Fix this

**Comment by GauchKirill** [comment]
I have already done

**Pull request merged**